osu.Framework/Platform/Window.cs

```diff
@@ -300,7 +316,12 @@ public Window(IWindowBackend windowBackend, IGraphicsBackend graphicsBackend)
             windowBackend.MouseWheel += OnMouseWheel;
             windowBackend.DragDrop += OnDragDrop;
 
+            windowBackend.DisplayChanged += d => CurrentDisplay.Value = d;
+
             graphicsBackend.Initialise(windowBackend);
+
+            CurrentDisplay.Value = windowBackend.CurrentDisplay;
+            CurrentDisplay.ValueChanged += evt => this.windowBackend.CurrentDisplay = evt.NewValue;
```

Comment:
```suggestion
            CurrentDisplay.ValueChanged += evt => windowBackend.CurrentDisplay = evt.NewValue;
```

Can probably be removed?